### DISCUSSION TITLE
Show difficulty number when hovering cursor over difficulty entry icon

### DISCUSSION THREAD

**Discussion by marioman63** [ppy/osu#26771] [Ideas] [answered]
Currently, if the player hovers the cursor over any of the difficulty icons on the main song entry, they can see precise difficulty numbers:
![image](https://github.com/ppy/osu/assets/76608775/0c8048f7-51b5-4e5c-a7be-fb261934855a)

However, if the user does the same on any of the difficulty entries, this does not occur:
![image](https://github.com/ppy/osu/assets/76608775/36e49f7b-d48c-4035-8e14-92bcaac7984a)

Feels like it would be a nice QoL and would make the functionality feel more consistent.

**Comment by Joehuu** [comment, marked as answer]
The star rating will be always visible on the panel in the new design, see https://github.com/ppy/osu/pull/22197.

**Reply by marioman63** [reply]
sweet. liking how colourful the new design is.
